Seneca-CDOT/starchart
# Add skeleton pages for user dashboard routes

Resolves #138 

Added basic pages for the following routes (as mentioned in the issue):
- `/domains`
- `/domains/new`
- `/domains/:id`
- `/certificate`
- `/certificate/instructions`

Also added a component for the header that will display on top of these pages. See image [here](https://github.com/Seneca-CDOT/starchart/pull/156#issuecomment-1421903942)

These are basic pages which can be styled later based on the [figma](https://www.figma.com/file/XAcnMIC1y4qumWinSalHI9/Starchart-UI?node-id=2105%3A2109&t=mPPiqAA5dItrRWGB-1) designs


### Steps to test
- Setup and run starchart locally
- Navigate to the above routes and check that they work. They will only have one simple heading text (irrelevant as this will be changed later anyway)
